src/network/msgpackobject.cc

```diff
@@ -156,6 +163,8 @@ void MsgpackObject::fromMsgpack(const msgpack::v2::object &obj) {
     value.ext.first = obj_ext.type();
     value.ext.second = std::make_shared<std::vector<uint8_t>>(obj_ext.data(), obj_ext.data()+obj_ext.size);
     break;
+  default:
```

Comment:
Doesn't compile on most platforms, since I added helper variable above (which can't be done if there is another label).